package.json

```diff
@@ -5,19 +5,20 @@
     "type": "git",
     "url": "https://github.com/mobify/split-test"
   },
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Yah that's kind of what I was getting at. I'm not sure though, like you said, it's not API changes, it's a build structure change. I feel like it would be strange if I was referencing ~0.1.2, got 0.1.3 and the file was missing. But if this were a highly used open source library I'd be hesitant to even go with 0.2.0 because it would still match ^0.1.2. Basically I don't want anybody to feel like they can npm install for bugfixes/features without issues and then encounter an issue like missing files. That would kind of throw me off as a user. It's idealistic I know. I would say it's fine as you have it unless we went with incrementung major. Sending this from dentist office kind of hard to type this lol